api/pkg/app/app.go

```diff
@@ -370,9 +391,41 @@ func configFileURL() (string, error) {
 	return val, nil
 }
 
+// initGhe looks for Github Enterprise url from the environment variables
+// and initialises the GHEConfig
+func initGhe() (*GHEConfig, error) {
+	var githubUrl, apiUrl, uploadsUrl string
+	var isGhe bool = false
```

Comment:
I guess we don't need to specify the value as false for a boolean variable, since it's already `false` by default